src/Database/Database.ts

```diff
@@ -9,7 +9,7 @@ import { DisplayToolEntry } from "./DataEntries/DisplayTool";
 import { DisplayWeaponEntry } from "./DataEntries/DisplayWeaponEntry";
 import { ArtifactDataManager } from "./DataManagers/ArtifactData";
 import { BuildResultDataManager } from "./DataManagers/BuildResult";
-import { BuildsettingDataManager } from "./DataManagers/BuildsettingData";
+import { BuildSettingDataManager } from "./DataManagers/BuildSettingData";
```

Comment:
You need to rename this file properly:
`mv BuildsettingData -> BuildsettingDatatmp`
`git add`
`mv BuildsettingDatatmp -> BuildSettingData`
`git add`